### ISSUE TITLE
Required Python versions not declared in package

### ISSUE THREAD

**Issue by andy-maier** [formencode/formencode#171]
While the package defines the trove classifiers for the supported Python versions, these classifiers are not used by pip, and the "python_requires" attribute that would be used by pip is not present in the package.

That causes pip to attempt to install FormEncode on Python versions that are not supported by FormEncode and then to fail in the Python version check in setup.py.

In addition, the version check in 1.3.1 is based on string comparison so it refuses to install on Python 3.10. This lead to this error in our own project where we have a defined minimum version for each package and test with that: https://github.com/pywbem/pywbem/runs/4392558860?check_suite_focus=true#step:11:234. The string based version checking has been fixed meanwhile, and I am just mentioning it because that is what triggered me to find the missing python_requires.

The `setup()` argument to be added to match the Python versions stated in the classifiers of the current master branch, would be:
```
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*',
```

**Comment by andy-maier** [comment]
Addressed in PR #172.

**Comment by andy-maier** [comment]
The issue has been addressed with a different PR #174. Closing the issue.
